sdk/client-core/core/src/main/java/io/clientcore/core/util/ClientLogger.java

```diff
@@ -46,43 +41,31 @@
  * @see Configuration
  */
 public class ClientLogger {
-    private final Logger logger;
-    private final String globalContextSerialized;
-    private static final char CR = '\r';
-    private static final char LF = '\n';
```

Comment:
Doesn't ClientLogger still need to remove new lines?

Reply:
nope, since we're json serializing, the serialization logic escapes new lines for us